tempest/api/workloadmgr/multiattach_volumes/test_multiattach_volumes.py

```diff
@@ -378,7 +386,7 @@ def test_01_multiattach_volumes(self):
                 raise Exception("Create workload with image booted vm")
 
             reporting.add_test_script(tests[1][0])
-            self.snapshot_id = self.workload_snapshot(self.wid, True)
+            self.snapshot_id = self.workload_snapshot(self.wid, True, snapshot_cleanup=True)
```

Comment:
default cleanup value is True, no need to explicitly pass that parameter again.

Reply:
updated